pkg/exporter/localmachine_exporter.go

```diff
@@ -0,0 +1,71 @@
+package exporter
```

Comment:
💡Thank you for this, here are my thoughts, I second Arnaud's thoughts here:  https://github.com/Azure/aks-periscope/pull/57#issuecomment-856746917 ; what is this local exporter achieving? 

Is it saving it into the machine the tool is run? (The consuming tool can do that like az cli can have some implementation at their end)  but regarding this tool If that is the case I don't think it will be that easy, the moment we are saving things into end-user's machine we have security and compliance issues, along with clean-up?

Reply:
Fair points, From periscope perspective, like Sophie mentioned the zip files are storage in container so that they can be pulled by clients like CLI. The folder where the Clients will Pull the zip files is client local problem. BTW, default is Azure storage, User has to explicitly specify the option if they wish to export the files locally, so the user it aware where and what files are exported on their machine.   If there is a security issue, let us know or point to compliance team so we can close on this aspect.

Regarding scenario, think of case where Pods has misconfigured enterprise proxy setting which is pretty common in Arc case. Azure blobs etc., will not be reachable in this case. Looping @shashankbarsin (Arc PM) on scenarios and questions.